src/dynamic_batch_scheduler.cc

```diff
@@ -141,6 +141,7 @@ DynamicBatchScheduler::Create(
 
   sched->scheduler_thread_exit_.store(false);
   if (dynamic_batching_enabled) {
+    sched->NewPayload();
```

Comment:
Do we need to call NewPayload() even when dynamic batching is disabled?

Reply:
@dyastremsky BatcherThread is not used when dynamic_batching is disabled. The payload is created by the main thread before directly being enqueued into the RateLimter. See L224-L230.